modules/core/client/components/Badge.js

```diff
@@ -0,0 +1,28 @@
+// External dependencies
+import PropTypes from 'prop-types';
+import React from 'react';
+import styled from 'styled-components';
+
+const Dot = styled.div`
+  background: #d9534f;
+  border-radius: 50%;
+  height: 5px;
+  position: absolute;
+  right: 10px;
+  top: 6px;
+  width: 5px;
+`;
```

Comment:
Maybe make it slightly bigger?

![image](https://user-images.githubusercontent.com/2955794/103154468-b52a1580-4797-11eb-8ac7-c048555abb15.png)

Anyway, looks good to me!

Reply:
Valid point! Made it a bit bigger.